index.html

```diff
@@ -50,6 +50,264 @@ <h1>I'm Matunda. Glad to see you!</h1>
         </div>
      </div>
     </section>
+    <!--Works Section-->
+    <section>
+      <div class="grid-container">
+        <div class="grid-card small-card">
+          <div class="snapshot">
+            <img src="./images/Tonic.svg">
+          </div>
+          <div class="project-details">
+            <div class="project-name"><h2>Tonic</h2></div>
+          
+            <div class="project-more-details">
+              <div class="client">
+                <p>CANOPY</p>
+                              
+              </div>
```

Comment:
[OPTIONAL]
- [ ] Same here too :point_up_2: you can add the `class="client"` to the `p` tag (i.g: `<p class="client">CANOPY</p>`)